src/api/spec/models/obs_factory/openqa_api_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe ObsFactory::OpenqaApi do
+  let(:openqa_api) { ObsFactory::OpenqaApi.new('https://some_url.com/') }
+
+  describe '::new' do
+    it { expect(openqa_api.base_url).to eq('https://some_url.com/api/v1/') }
+  end
+
+  describe '#get' do
+    context 'when at least one parameter key is empty' do
+      it { expect(openqa_api.get('/foo/bar', param1: '1', param2: nil)).to eq({}) }
```

Comment:
I would use ```be_empty``` instead of ```eq({})```

Reply:
But be empty would also match an empty array or string